### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by danmcp**

### CODE REVIEW

**Comment by danmcp** [comment]
@Erin-Donnelly I would love your input on the content here.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
-# radiology
-Collaboration with Boston Children's Hospital, Boston University, and Red Hat to improve the scale and performance of the ChRIS project
+# Radiology
+The radiology project is a collaboration with Boston Children's Hospital, Boston University, and Red Hat to improve the scale and performance of [ChRIS](https://github.com/FNNDSC/).
+
+## ChRIS
+ChRIS (Children’s Research Integration System) is a web-based medical image platform that allows for various forms of medical image (Ex: MRIs) processing. ChRIS itself is comprised of multiple open source projects hosted on [GitHub](https://github.com/FNNDSC/) with the intention to make the research and capabilities available to other hospitals and institutions.
```
Comment (Erin-Donnelly) [inline README.md:5]: "...various forms of medical image (Ex: MRIs) processing." 
--> 
"...various forms of medical image processing (for example, MRIs)."

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+
+## Massachusetts Open Cloud
+The [MOC](https://massopen.cloud/) is a cloud computing environment built in collaboration with 5 Massachusetts area universities (Boston University, Harvard, MIT, Northeastern, UMass) used primarily for educational and research purposes. The MOC, however, has the goal of expanding its footprint/usage by providing vertical solutions, such as the ChRIS project, to other institutions with similar needs.
```
Comment (Erin-Donnelly) [inline README.md:8]: "5" --> "five"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,24 @@
...
+
+## Goals
+The desire of Boston Children’s Hospital, Boston University and Red Hat is to help improve the scale and efficiency of the ChRIS platform using the MOC and various Red Hat technologies including OpenShift and OpenStack. Today, processing a single set of images takes ~10 hrs. The goal is to get that down to a few mins to give the patients immediate feedback.
```
Comment (Erin-Donnelly) [inline README.md:11]: Boston Children’s Hospital, Boston University, and Red Hat aim to improve the scale and efficiency of the ChRIS platform using the MOC and various Red Hat technologies, including OpenShift and OpenStack. Today, processing a single set of images takes about ten hours. The goal is to get that down to a few minutes to give patients quicker feedback.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,24 @@
...
+
+## Architecture
+ChRIS is comprised of several components/services and designed to have multiple heterogeneous clusters available for image processing. In the example shown below, the datacenter providing the image processing is the MOC which is backed by OpenShift running on OpenStack. The primary interfaces into the MOC are pman and pfioh which provide the API for pfcon to communicate with where pfioh handles the input/output and pman handles the image processing requests. The image plugins themselves run as container jobs on OpenShift.
```
Comment (Erin-Donnelly) [inline README.md:14]: ChRIS is comprised of several components and services, and it is designed to have multiple heterogeneous clusters available for image processing. In the example shown below, the data center providing the image processing is the MOC, which is backed by OpenShift running on OpenStack. The primary interfaces into the MOC are pman and pfioh which provide the API for pfcon to communicate with where pfioh handles the input/output and pman handles the image processing requests. The image plugins themselves run as container jobs on OpenShift.

- I'm a little confused by the wording of the second to last sentence. Can it be worded "The primary interfaces into the MOC are pman and pfioh. These interfaces provide the API for pfcon to communicate with the MOC, where pfioh handles the input/ouput, and pman handles the image processing requests." It's a little more wordy, but I think a bit more clear.

**Review by Erin-Donnelly — COMMENTED** [review]
Sorry this took so long! (Been a bit in the weeds with Ceph.)

So this is mostly just nit-picky style things (per Red Hat's style guide), the content was super clear already. If you have any areas of concern or anything else let me know!

**Comment by danmcp** [comment]
@Erin-Donnelly Do you want to submit a pull request with these changes?

**Comment by Erin-Donnelly** [comment]
@danmcp sure! All changes look good to you?

**Comment by danmcp** [comment]
> @danmcp sure! All changes look good to you?

@Erin-Donnelly Yes

**Pull request merged**